draft-ietf-quic-transport.md

```diff
@@ -1461,127 +1462,215 @@ connection if the integrity check fails with a PROTOCOL_VIOLATION error code.
+initiate to confirm the client's ownership of its address (see
+{{migration-server}}).
+
+
+#### Verifying the PMTU
+
+When the client intends to use a new local address, it MAY establish server
+reachability from the new address, and it SHOULD verify the new path's MTU.  To
+achieve these ends, the client sends a packet containing a PATH_CHALLENGE frame
+from this new address before sending any other packets.  The PATH_CHALLENGE
+frame MAY be bundled with other frames, but this packet MUST be padded to
+exactly 1200 octets unless the client has a reasonable assurance that the PMTU
+is larger.  Sending a packet of this size confirms that the network path from
+the client to the server supports an MTU of this size.
+
+A client MAY send additional PATH_CHALLENGE frames to handle packet loss or to
+get more measurements on the new network path, subject to the congestion
+controller's limits.
+
+The client SHOULD use fresh random data in every PATH_CHALLENGE frame so that it
+can perform more accurate measurements by associating the server's response with
+the causative PATH_CHALLENGE.
+
+When the client receives a PATH_RESPONSE from the server and verifies that the
+included data matches that sent by the client in a previously sent
+PATH_CHALLENGE, the server is considered reachable from the new address and the
+PMTU of the new network path is considered verified.
+
+A client SHOULD abandon the new address if it does not receive a PATH_RESPONSE
+after sending some number of PATH_CHALLENGE frames and/or after some time has
+passed. Note that the client may be receiving data from the server during this
+period, but not receiving a PATH_RESPONSE may be indicative of a PMTU detection
+failure.
+
+
+#### Responding to Server's Validation
+
+A client may receive a PATH_CHALLENGE from a server that is validating the
+client's ownership of its address, as described in {{migration-server}}. The
+client MUST respond to this frame by echoing the data from the server's
+PATH_CHALLENGE frame in a PATH_RESPONSE frame.
+
+A client's PATH_RESPONSE MAY be bundled with other frames and does not require
+additional padding.
+
+A client MAY send more than one PATH_RESPONSE in response to a PATH_CHALLENGE,
+as allowed by the congestion controller, to ensure that packet loss does not
+unduly delay or cause failure of the server's validation process.
+
+A client may trigger the server's validation unwittingly when its public address
+changes, such as when a NAT rebinds the client's private address to a new public
+address.  Though the client may be oblivious of such an address change, it MUST
+respond to the consequent validation.  Receiving a PATH_CHALLENGE from the
+server when the client is not actively changing addresses may be indicative of
+other network activity causing its public address to change, or it may be
+indicative of a server choosing to validate a client's address after a period of
+quiescence.
+
+
+### Server Behavior {{migration-server}}
```

Comment:
Shouldn't this be `{#migration-server}`?

Reply:
sigh, yes. fixed.